src/libratbag-enums.h

```diff
@@ -129,6 +129,16 @@ enum ratbag_resolution_capability {
 	 * The resolution can be set for x and y separately.
 	 */
 	RATBAG_RESOLUTION_CAP_SEPARATE_XY_RESOLUTION = 1,
+
+	/**
+	 * The resolution can be disabled and enabled.
+	 * 
+	 * Note that this capability only notes the general capability. A
+	 * specific resolution may still fail to be disabled, e.g. when it
+	 * is the active resolution on the device.
+	 * 
+	 */
+	RATBAG_RESOLUTION_CAP_DISABLE = 3, // <- what should be the abi identifier? Needed for piper
```

Comment:
Does the value actually matter? Like this it works (although I can't test `RATBAG_RESOLUTION_CAP_SEPARATE_XY_RESOLUTION`).

Reply:
I think it's the bitshift again? Then a value of 2 would make more sense...

Reply:
Yes, it's bitshift (see `ratbag_resolution_set_cap`). The value should be 2. Doesn't make sense to jump from 1 to 3.